service/src/main/java/bio/terra/workspace/service/crl/CrlService.java

```diff
@@ -231,6 +237,34 @@ public MonitorManager getMonitorManager(
         .authenticate(azureCreds, azureProfile);
   }
 
+  public ContainerServiceManager getContainerServiceManager(
+      AzureCloudContext azureCloudContext, AzureConfiguration azureConfig) {
+    assertCrlInUse();
+    final var azureCreds = getManagedAppCredentials(azureConfig);
+    final var azureProfile = getAzureProfile(azureCloudContext);
+    return bio.terra.cloudres.azure.resourcemanager.common.Defaults.crlConfigure(
+            clientConfig, ContainerServiceManager.configure())
+        .authenticate(azureCreds, azureProfile);
+  }
+
+  public PostgreSqlManager getPostgreSqlManager(
+      AzureCloudContext azureCloudContext, AzureConfiguration azureConfig) {
+    assertCrlInUse();
+    final var azureCreds = getManagedAppCredentials(azureConfig);
+    final var azureProfile = getAzureProfile(azureCloudContext);
+    // PostgreSqlManager.configure does not return the right type, so inline
```

Comment:
I think there was that situation for another resource type (batch) and CRL defined a `Defaults.crlConfigure` in the `batch` package: https://github.com/DataBiosphere/terra-cloud-resource-lib/blob/master/src/main/java/bio/terra/cloudres/azure/resourcemanager/batch/Defaults.java#L25